src/system/SearchModule/Api/UserApi.php

```diff
@@ -229,11 +238,11 @@ public function getall($args)
         if (!isset($args['numitems']) || !is_numeric($args['numitems'])) {
             $args['numitems'] = -1;
         }
-        if (!isset($args['sortorder']) || !in_array($args['sortorder'], array('count', 'date'))) {
+        if (!isset($args['sortorder']) || !in_array($args['sortorder'], ['count', 'date'])) {
             $args['sortorder'] = 'count';
         }
 
-        $items = array();
+        $items = [];
 
         // Security check
         if (!SecurityUtil::checkPermission('ZikulaSearchModule::', '::', ACCESS_OVERVIEW)) {
```

Comment:
IMO, all security checks should be done at the Controller level (and use shortcut to new service).